ocs_ci/utility/reporting.py

```diff
@@ -48,6 +48,7 @@ def get_rp_launch_attributes():
     rp_attrs["worker_instance_type"] = config.ENV_DATA.get("worker_instance_type")
     rp_attrs["ocp_version"] = get_ocp_version()
     rp_attrs["ocs_version"] = config.ENV_DATA.get("ocs_version")
+    rp_attrs["ocs_build"] = get_ocs_build_number()
```

Comment:
Is this triggered at the start of the execution? Or once we have ocs running?

Mainly asking about deployment scenario.

As I see you call get client function, just not sure why we don't have client yet, maybe in deployment start, hence asking.

Reply:
Yeah this is the main issue with this approach. I originally hit the issue where we didn't have the client so i added this call to ensure we do. However the real problem is that this entire process is called from within an autouse fixture prior to any deployment being done, so we won't be able to do things like [pull CSV data](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L1500) anyways since it doesn't exist. 

In order to utilize `get_ocs_build_number` we will need to introduce some other place (post deployment) where we can call this. However, I haven't tested whether we can update the testsuite custom properties at that time.